README.md

```diff
@@ -3377,3 +3377,20 @@ What's up, guys?
 
 ## Test
 That my test
+
+# Kottans-frontend
+
+<p> Hello! My name is Tania. I want to be a Frontend developer. I hope that this course helps me to achieve the goal. </p>
```

Comment:
Avoid using HTML tags in markdown. Use [this resource](https://www.markdownguide.org/cheat-sheet/) for updating your changes

Reply:
Hello,
And now? I tried to fix it and pushed again. 
Thank you for the information about markdown, it was new for me)
If I correct understand, tag <p> we can use in markdown too?